boards/riscv/dt_bl10_devkit/dt_bl10_devkit.dts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, ATL Electronics
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "dt_bl10_devkit-pinctrl.dtsi"
+
+/ {
+	model = "2.4GHz Wi-Fi and BLE coexistence Module Development Kit";
+	compatible = "bflb,bl602";
+
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,itcm = &itcm;
+		zephyr,dtcm = &dtcm;
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <160000000>;
+};
+
+&spi1 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0x4000b000 0x1000 0x23000000 0xc00000>;
+	flash0: flash@0 {
+		compatible = "issi,is25lp128", "jedec,spi-nor";
+		size = <134217728>;
+		label = "FLASH0";
+		jedec-id = [96 60 18];
+		reg = <0>;
+		// Dummy entry
+		spi-max-frequency = <0>;
```

Comment:
What's going on here? This does not look correct. Is it just for draft?

Reply:
Hi @mbolivar-nordic , all this PR still marked as draft. I only asked to @gmarull some tips about pinctrl because it is a new API and this SoC have some odd parts.